.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": " Codespace",
+	"image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0.202.3-16",
+	"settings": {
+		"workbench.colorTheme": "Default Dark+"
+	},
+	"extensions": [
+		"eamodio.gitlens",
+		"cschleiden.vscode-github-actions",
+		"redhat.vscode-yaml",
+		"bierner.markdown-preview-github-styles",
+		"jmrog.vscode-nuget-package-manager",
+		"coenraads.bracket-pair-colorizer",
+		"vscode-icons-team.vscode-icons",
+		"editorconfig.editorconfig",
```

Comment:
If we are not also including an .editorconfig with this PR, why is this extension needed?

Reply:
We included, just because it is a popular choice with GitHub projects and thought that it might be useful in the future and ready-to-go